features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/ListRoles.jsx

```diff
@@ -40,10 +40,13 @@ const headCells = [
         numeric: false, disablePadding: false, label: 'Roles',
     },
     {
-        id: 'permissions', numeric: false, disablePadding: false, label: 'Permissions',
+        id: 'permissions', numeric: false, disablePadding: false, label: 'Scope Assignments',
     },
 ];
 
+const pageDesc = 'Scope assignments are related to internal APIM specific scope assignments and are not related'
+                + ' to role permission assignments in the management console.';
```

Comment:
Scope assignments are only related to internal, APIM-specific scope assignments. They are not related to role permission assignments in the Management Console.

Reply:
Fixed with 79a0036